app/components/files/file.tsx

```diff
@@ -45,7 +45,6 @@ type FileProps = {
 const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
     return {
         fileWrapper: {
-            flex: 1,
```

Comment:
curious about the removal of flex here and the other places, can you expand why are we removing it ?

Reply:
When applying max-height to the container that wraps the file container. The file container shrinks and gets squeezed if it has the property `flex: 1`. This happens due to the react-native Yoga layout engine. 

<img width="324" height="376" alt="Screenshot 2025-09-07 at 7 07 49 PM" src="https://github.com/user-attachments/assets/1fec560b-fc9a-4aa5-bd13-55465355de30" />

Without `flex: 1`:

<img width="335" height="546" alt="Screenshot 2025-09-07 at 7 08 23 PM" src="https://github.com/user-attachments/assets/59b509c1-2cff-4ba5-bfc6-92661bf9e75e" />

Reply:
And what about files in a regular post, what about tablets?